src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -115,7 +117,8 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
           <Text
             style={{
               fontSize: 14,
-              color: COLORS.white,
+              fontFamily: fonts.inter[400],
+              color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fontFamily` property has been added to the style object for the Text component, and the `color` property has been changed. This improves consistency in typography and color scheme across the application.
```diff
- color: COLORS.white,
+ fontFamily: fonts.inter[400],
+ color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->